scanpipe/api/serializers.py

```diff
@@ -144,7 +144,7 @@ def create(self, validated_data):
             project.add_input_file(upload_file)
 
         if pipeline:
-            project.add_pipeline(pipeline, start_run=bool(upload_file))
+            project.add_pipeline(pipeline, start=bool(upload_file))
```

Comment:
Nit: I wonder if start if the proper name to use there: what about `run` or `launch` or `execute` ? e.g. a pipeline has a start and end step, but the action to execute may not be best described as "starting" which is state more than an action?

Reply:
`Run` is the current name of the Run that wrap a pipeline execution, so "run a run" is not great.
Execute is the action on the Pipeline class to execute each step.

> a pipeline has a start and end step
This is not the case anymore.

So the idea here is to "start" a Run which is a bit more (celery task management) than a Pipeline execution.

Reply:
After discussion, we decided to use `execute` instead of `start`.